### DISCUSSION TITLE
Is it possible to change some output on Main dasboard as pe below:

### DISCUSSION THREAD

**Discussion by kronicklez** [akvorado/akvorado#1429] [Q&A]
Hi vincent / expert,

First of all i'm using docker method. To change some default value on visualization page then we just copy some command in the "docker compose exec akvorado-orchestrator akvorado orchestrator --check --dump /etc/akvorado/akvorado.yaml" then put in the console.yaml. But how if we want to change some default value on the main pages dashboard as per below:

1.) 
![image](https://github.com/user-attachments/assets/9fa6d2b8-fe9d-4f60-94d9-22290729e451)

Default it just show 6 top value. How we can change to other value such as 8 or 10?

2.) How to change icon login and text for this below? I'm asking because my management asking whether it can be change or not to follow company policy. 
![{2A84C3C6-16B1-4615-8E42-49D7151A5389}](https://github.com/user-attachments/assets/bb4022fc-fdfd-4542-9cee-260fb2a0559e)

Lastly i can say this application that u develop is very2 good for small and medium ISP. Appreciate your guide regarding two question above.

**Comment by vincentbernat** [comment]
1. This is not configurable.
2. You need to plug your own authentication system with an authenticating proxy and then override the `/api/v0/console/user/avatar` with the icon you want.

**Comment by luckman212** [comment]
As a followup for question 1, is it still suggested to use Grafana if a custom dashboard is desired? And we still use Akvorado as the NetFlow collector?
